### PR TITLE
fix:trim prefix for node names

### PR DESCRIPTION
**Pull request by Despire**
Fixes the issue where when the changeApiEndpoint playbook would reference the old node names with the prefix clusterName + clusterHash.

### CODE REVIEW

File: services/ansibler/server/server.go [hunk 1]
```diff
@@ -97,7 +99,7 @@ func (*server) TeardownLoadBalancers(ctx context.Context, req *pb.TeardownLBRequ
 	for _, lb := range req.DeletedState.LoadBalancerClusters {
 		np, ok := k8sNodepools[lb.TargetedK8S]
 		if !ok {
-			log.Error().Msgf("LoadBalancer %s has not found a target k8s cluster %s", lb.ClusterInfo.Name, lb.TargetedK8S)
+			log.Info().Msgf("LoadBalancer %s has not found a target k8s cluster %s", lb.ClusterInfo.Name, lb.TargetedK8S)
```
Comment (MiroslavRepka) [inline services/ansibler/server/server.go:102]: Isn't this considered an error/warning? If we have a Loadbalancer in a state which does not target any Kubernetes cluster, it should be caught by the frontend, thus if we detect it here, something went wrong.

File: services/ansibler/server/server.go [hunk 2]
```diff
@@ -156,7 +161,7 @@ func (*server) SetUpLoadbalancers(_ context.Context, req *pb.SetUpLBRequest) (*p
 	for _, lb := range req.DesiredState.LoadBalancerClusters {
 		np, ok := k8sNodepools[lb.TargetedK8S]
 		if !ok {
-			log.Error().Msgf("Loadbalancer %s has not found a target k8s cluster (%s)", lb.ClusterInfo.Name, lb.TargetedK8S)
+			log.Info().Msgf("Loadbalancer %s has not found a target k8s cluster (%s)", lb.ClusterInfo.Name, lb.TargetedK8S)
```
Comment (MiroslavRepka) [inline services/ansibler/server/server.go:164]: Isn't this considered an error/warning? If we have a Loadbalancer in a state which does not target any Kubernetes cluster, it should be caught by the frontend, thus if we detect it here, something went wrong.

File: services/ansibler/server/server.go [hunk 3]
```diff
@@ -78,12 +78,14 @@ func (*server) TeardownLoadBalancers(ctx context.Context, req *pb.TeardownLBRequ
 		attached        = make(map[string]bool)           // [k8sClusterName]Bool
 		k8sNodepools    = make(map[string][]*pb.NodePool) //[k8sClusterName][]nodepools
 		k8sNodepoolsKey = make(map[string]string)         //[k8sClusterName]keys
+		clusterIDs      = make(map[string]string)         //[k8sClusterName]prefix (clusterName + hash)
```
Comment (MiroslavRepka) [inline services/ansibler/server/server.go:81]: Just for the consistency
`//[k8sClusterName]prefix (clusterName + hash)`  -> `//[k8sClusterName]clusterID (<clusterName>-<hash>)`

File: services/ansibler/server/server.go [hunk 4]
```diff
@@ -139,13 +142,15 @@ func (*server) SetUpLoadbalancers(_ context.Context, req *pb.SetUpLBRequest) (*p
 		lbInfos         = make(map[string]*LBInfo)        //[k8sClusterName]lbInfo
 		k8sNodepools    = make(map[string][]*pb.NodePool) //[k8sClusterName][]nodepools
 		k8sNodepoolsKey = make(map[string]string)         //[k8sClusterName]keys
+		clusterIDs      = make(map[string]string)         //[k8sClusterName]prefix (clusterName + hash)
```
Comment (MiroslavRepka) [inline services/ansibler/server/server.go:145]: Just for the consistency
`//[k8sClusterName]prefix (clusterName + hash)`  -> `//[k8sClusterName]clusterID (<clusterName>-<hash>)`

**Review by MiroslavRepka — CHANGES_REQUESTED** [review]
LGTM, please see my comments. Thanks, @Despire for spotting the issue!

**Comment by Despire** [comment]
Changes implemented, also changed to `inventory_hostname` which actually will use the name from the inventory file.

**Review by MiroslavRepka — APPROVED** [review]

**Pull request merged**